sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/clienttelemetry/ClientTelemetryMetrics.java

```diff
@@ -985,6 +989,26 @@ private void recordStoreResponseStatistics(
                     requestCounter.increment();
                 }
 
+                if (actualItemCount > 0) {
```

Comment:
condition should be removed - see comment below

Reply:
updated